make/mt-os/mt-cc3200.mk

```diff
@@ -5,41 +5,59 @@
 
 include make/mt-os/mt-os-common.mk
 
-###
-## COMPILER NAME
-###
-COMPILER ?= arm_15.12.3.LTS
+ifdef XI_TRAVIS_BUILD
+	# For Xively C Client Continuous Integration only.
+	COMPILER ?= arm_15.12.3.LTS
```

Comment:
maybe this is a separate task / story: to update travis' CC3200 compiler, so I guess we can skip this here